docs/docs/Develop/logging.mdx

```diff
@@ -9,46 +9,86 @@ import TabItem from '@theme/TabItem';
 
-Langflow uses the `loguru` library for logging.
+Langflow uses the [structlog](https://www.structlog.org) library for logging.
 
 The default logfile is named `langflow.log`.
 Langflow also produces logfiles for flows.
 
+Log files are stored in JSON format with structured metadata.
+
 ## Log storage
 
-The default logfile storage location depends on your operating system:
+Langflow logs are stored in the config directory specified in the `LANGFLOW_CONFIG_DIR` environment variable.
+The default config directory location depends on your operating system and installation method:
 
-- **macOS Desktop**:`/Users/<username>/.langflow/cache`
+- **macOS Desktop**:`/Users/<username>/Library/Logs/com.Langflow`
+- **macOS OSS**:`/Users/<username>/Library/Caches/langflow`
 - **Windows Desktop**:`C:\Users\<username>\AppData\Roaming\com.Langflow\cache`
-- **OSS macOS/Windows/Linux/WSL (`uv pip install`)**: `<path_to_venv>/lib/python3.12/site-packages/langflow/cache`
-- **OSS macOS/Windows/Linux/WSL (`git clone`)**: `<path_to_clone>/src/backend/base/langflow/cache`
 
 To customize log storage, see [Configure log options](#configure-log-options).
 
-## Configure log options
+### Configure log storage options
 
 Use [Langflow environment variables](/environment-variables) to configure logging options in your Langflow `.env` file, and then start Langflow with `uv run langflow run --env-file .env`.
 
 The following environment variables are available to configure logging:
 
+* `LANGFLOW_CONFIG_DIR`: Sets the config directory where logs and cache files are stored. For example, `LANGFLOW_CONFIG_DIR=/path/to/config`.
+
 * `LANGFLOW_LOG_LEVEL`: Sets the log level as one of `DEBUG`, `ERROR` (default), `INFO`, `WARNING`, and `CRITICAL`. For example, `LANGFLOW_LOG_LEVEL=DEBUG`.
 
 * `LANGFLOW_LOG_FILE`: Sets the log file storage location if you want to use a non-default location.
 For example, `LANGFLOW_LOG_FILE=path/to/logfile.log`.
 
 * `LANGFLOW_LOG_ENV`: Controls log output and formatting.
 
-    * `LANGFLOW_LOG_ENV=container`: Outputs serialized JSON to stdout.
+    * `LANGFLOW_LOG_ENV=container` or `LANGFLOW_LOG_ENV=container_json`: Outputs serialized JSON to stdout.
     * `LANGFLOW_LOG_ENV=container_csv`: Outputs CSV-formatted plain text to stdout.
     * `LANGFLOW_LOG_ENV=default` or unset: Outputs prettified output with [RichHandler](https://rich.readthedocs.io/en/stable/reference/logging.html).
 
-* `LANGFLOW_LOG_ROTATION` controls when the log file is rotated, either based on time or file size. By default, logs are rotated every 1 day.
+A complete example `.env` file with logging values is available in the [Langflow repository](https://github.com/langflow-ai/langflow/blob/main/.env.example).
+
+## View logs in real-time
+
+To monitor Langflow logs as they are generated, you can use the `tail -f` command to follow the log file:
+
+1. Change to your [Langflow config directory](#log-storage):
+
```

Comment:
You have macOS and Windows here, but (based on the paths on lines 21-26) you're using the macOS _OSS_ path and the Windows _desktop_ path?